feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/sync/RealExternalAccountsSyncService.kt

```diff
@@ -300,7 +298,7 @@ internal class RealExternalAccountsSyncService @Inject constructor(
                     return@controllersLoop
                 }
 
-                val existingControlledAccounts = existingAccountsByParentId[controller.id].orEmpty()
+                val existingControlledAccounts = controllersByParentId[controller.id].orEmpty()
```

Comment:
Can you please describe a case when this change matters? Am I right that this fixes dublication in case some multisig is present twice in the list and when we process is second time, we dont see the account added in the first iteration since `existingAccountsByParentId` only contain snapshot of the db accounts before the processing start?

If so, can you please describe the case itself, like what is the structure of accounts e.t.c. that causes the same account to be present multiple times in the `foundExternalAccounts`